README.md

```diff
@@ -1,73 +1,36 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Hell-Microverse-World
 
-> Description the project.
+This is my first project as a student at Microverse. It is a simple project built in HTML and CSS and it includes linters.
 
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+This project is built using the following languages.
+- HTML
+- CSS
 
 ## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
```

Comment:
Please explain the procedure or steps to follow if one wants to get a copy of your project up and running. It helps fellow developers or anyone to easily test the project.